visual_marker_mapping/src/main_detection.cpp

```diff
@@ -92,12 +98,28 @@ int main(int argc, char* argv[])
             }
         }
 
+        bool use_umich = false;
```

Comment:
Ich glaube du brauchst den boolean nicht. Du kannst direkt den string Vergleich in das if weiter unten verschieben.